src/Plugin/Replace/Handler.php

```diff
@@ -57,106 +80,103 @@ public function run(): Task {
 	}
 
 	/**
-	 * @param  Client  $manticoreClient
-	 * @param  string  $table
-	 * @return array <int, array<string, string>>
+	 * @param Client $manticoreClient
+	 * @param string $table
+	 * @return array<int|string, array{type:string, properties:string}>
 	 * @throws ManticoreSearchClientError
 	 */
 	private static function getFields(Client $manticoreClient, string $table): array {
 		$descResult = $manticoreClient
-			->sendRequest('DESC '.$table)
+			->sendRequest('DESC ' . $table)
 			->getResult();
 
 		if (is_array($descResult[0])) {
-			return $descResult[0]['data'];
+			$fields = [];
+			/** @var array{Type:string, Properties:string, Field:string} $field */
+			foreach ($descResult[0]['data'] as $field) {
+				$fields[$field['Field']] = ['type' => $field['Type'], 'properties' => $field['Properties']];
+			}
+
+			return $fields;
 		}
 
-		return [];
+		throw ManticoreSearchClientError::create('Table hasn\'t fields');
 	}
 
 
 	/**
-	 * @param  array<int, array<string, string>>  $fields
+	 * @param array<int|string, array{type:string, properties:string}> $fields
 	 * @return void
-	 * @throws \Manticoresearch\Buddy\Core\Error\GenericError
+	 * @throws GenericError
 	 */
 	private static function checkStoredFields(array $fields): void {
-		foreach ($fields as $field) {
-			if ($field['Type'] !== 'text'
-				|| str_contains($field['Properties'], 'stored')) {
+		foreach ($fields as $fieldName => $fieldSettings) {
+			if ($fieldSettings['type'] !== 'text'
+				|| str_contains($fieldSettings['properties'], 'stored')) {
 				continue;
 			}
 
 			ManticoreSearchResponseError::throw(
-				'Field '.$field['Field']
-				.' doesn\'t have stored property. Replace query can\'t be performed'
+				'Field ' . $fieldName
+				. ' doesn\'t have stored property. Replace query can\'t be performed'
 			);
 		}
 	}
 
 	/**
-	 * @param  Client  $manticoreClient
-	 * @param  Payload  $payload
-	 * @param  array<int, array<string, string>>  $fields
-	 * @return array<string, string|int>
+	 * @param Client $manticoreClient
+	 * @param Payload $payload
+	 * @param array<int|string, array{type:string, properties:string}> $fields
+	 * @return array<string, bool|float|int|string>
 	 * @throws ManticoreSearchClientError
 	 */
 	private static function getRecordValues(Client $manticoreClient, Payload $payload, array $fields): array {
-		$sql = 'SELECT * FROM '.$payload->table.' WHERE id = '.$payload->id;
+		$sql = 'SELECT * FROM ' . $payload->table . ' WHERE id = ' . $payload->id;
```

Comment:
When we can it's a good idea to avoid concatenation and use interpolation:
```php
"select * from {$payload->table} where id = {$payload->id}"
```

for better readability